src/trafficSplitter/getRandomEnvName.ts

```diff
@@ -12,7 +12,7 @@ const chance = new Chance();
  * the `beta` environment. However, the order of the environments assigned to
  * one set of 5 sessions may differ from the next 5 sessions.
  */
-export function* createEnvNameGenerator(
+export function* createRandomEnvNameGenerator(
```

Comment:
This isn't really "random"

Reply:
The choice will be random within the constraints of the defined weights. Although every 5 requests will include 4 matter and 1 beta, the order of these is random because we are using `chance.pickone`.